database/src/test/kotlin/no/nav/su/se/bakover/database/søknad/SøknadPostgresRepoTest.kt

```diff
@@ -35,4 +36,22 @@ internal class SøknadPostgresRepoTest {
             }
         }
     }
+
+    @Test
+    fun `slettet behandling for søknad skal ikke bli hentet`() {
+        withMigratedDb {
+            EmbeddedDatabase.instance().withSession {
```

Comment:
Tror ikke det er behov for en ekstra session her.

Reply:
👍